tests/e2e_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  "rules": {
+    "prettier/prettier": 2,
```

Comment:
I heard of:

```js
"prettier/prettier": ["error"]
```

what does `2` do?

Reply:
"off" or 0 - turn the rule off
"warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
"error" or 2 - turn the rule on as an error (exit code is 1 when triggered)

Reply:
could we be explicit? it is not easy to find what the numbers mean.

Reply:
? https://eslint.org/docs/latest/user-guide/configuring/rules - this is an eslint config file after all...

Reply:
on second thought, you're right - changed it in the outer config!